aws-lc-fips-sys/builder/bindgen.rs

```diff
@@ -62,12 +62,12 @@ fn prepare_clang_args(manifest_dir: &Path, build_prefix: Option<&str>) -> Vec<St
     clang_args
 }
 
-const COPYRIGHT: &str = r#"
```

Comment:
Were these to get around clippy warnings?

Reply:
Yeah, clippy started complaining about this.